### PR TITLE
fix: add takanome-dev

### LINKED ISSUE #1113: Takanome Dev
## Profile Screenshot

github-screen

### PR DESCRIPTION
**Pull request by TAKANOME-DEV**
resolves #1113 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes
- `takanome-dev.json` file has been added

### CODE REVIEW

File: profiles/takanome-dev.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://camo.githubusercontent.com/2d8017444f17e2eb9cd2b304b3437de0efdc887543fb9f90f3e0942d608ef776/68747470733a2f2f7265732e636c6f7564696e6172792e636f6d2f646166777a736f64302f696d6167652f75706c6f61642f76313635313933343438382f6769746875622d73637265656e5f706e6763346b2e706e67",
+  "issueId": 1113,
+  "name": "Takanome Dev",
+  "username": "takanome-dev"
+}
```
Comment (Panquesito7) [inline profiles/takanome-dev.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krshkodes — CHANGES_REQUESTED** [review]
After Panquesito's suggestion, it should be good!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Readable, Not maintainable

**Comment by TAKANOME-DEV** [comment]
Glad to hear from you, endline added. Thanks

**Comment by TAKANOME-DEV** [comment]
@Panquesito7,  all changes have been made ☺️

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**